librina/include/librina/common.h

```diff
@@ -616,6 +623,21 @@ class SerializedObject {
         void initialize(const SerializedObject& other );
 };
 
+struct UcharArray {
```

Comment:
The problem is that serialized object is a char array, and I needed unsigned char array to interact with the openssl APIs. Maybe we can change all the serialized object stuff by UCharArray, but I wasn't sure and it was a lot of changes. Let's see if a SerializedObject (which is really a class wrapping a char array) can be updated to unsigned char array (in fact, to represent a raw byte the usual thing to use is an unsigned char, not a char, right?)